improvements to github exporter docs (#166)
port-labs/port-docs: docs/exporters/bitbucket-exporter/exporting-files.md

:::

# Exporting files

> By the end of this tutorial, we will have a `microservice` Blueprint that contains an auto-synced README.md file into a Port property.

Reviewer: ```suggestion
In this tutorial you will learn how to create a `microservice` Blueprint that contains an auto-synced README.md file into a Port property.
```
